docs/install-and-run.md

```diff
@@ -1,208 +1,239 @@
 # Install and run
 
 This document shows you how to install EarlGrey and then how to set up and run your first
-test.
-
-## Prerequisites
-
-For EarlGrey to function properly, make sure the app under test has correct launch screen images present for all supported devices (see [iOS Developer Library, Launch Files](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html)).
+test.M
 
 ## Install EarlGrey
 
-You can add EarlGrey to Xcode projects in two ways: using CocoaPods, or as a framework. If you wish to contribute to the EarlGrey project, the **EarlGrey.xcodeproj** generated from the [GitHub Installation](#github-installation) section is integrated with the Unit and Functional Test projects for you to work with the source.
+You can add EarlGrey to Xcode projects in three ways: using [CocoaPods](#cocoapods-installation), [Carthage](#carthage-installation) or [Manually](#github-installation) through the Xcode Project.
 
-### CocoaPods installation
+For EarlGrey, we highly recommend [CocoaPods](http://cocoapods.org/pods/EarlGrey) as the best way to get started.
```

Comment:
The cocoapods link should point to the section inside this file for Cocoapods installation